compiler/x/codegen/ControlFlowEvaluator.cpp

```diff
@@ -524,103 +524,98 @@ TR::Register *OMR::X86::TreeEvaluator::tableEvaluator(TR::Node *node, TR::CodeGe
-   startLabel->setStartInternalControlFlow();
-   doneLabel->setEndInternalControlFlow();
-   generateLabelInstruction(LABEL, node, startLabel, cg);
-
-   if (cg->canClobberNodesRegister(child))
-      {
-      trgReg = reg; // use first child as a target
-      }
-   else
-      {
-      switch (data_type)
-         {
-         case TR::Int32:
-            trgReg = cg->allocateRegister();
-            break;
-         case TR::Int64:
-            if (two_reg)
-               trgReg = cg->allocateRegisterPair(cg->allocateRegister(), cg->allocateRegister());
-            else
-               trgReg = cg->allocateRegister();
-            break;
-         default:
-            TR_ASSERT(false, "assertion failure");
-         }
-      if (two_reg)
-         {
-         generateRegRegInstruction(move_op, node, trgReg->getLowOrder(), reg->getLowOrder(), cg);
-         generateRegRegInstruction(move_op, node, trgReg->getHighOrder(), reg->getHighOrder(), cg);
-         }
-      else
-         {
-         generateRegRegInstruction(move_op, node, trgReg, reg, cg);
-         }
-      }
-
-   child = node->getChild(1);
-   reg = cg->evaluate(child);
-   TR::RegisterDependencyConditions *dep;
-   if (two_reg)
-      {
-      TR::LabelSymbol *gtLabel = generateLabelSymbol(cg);
-      generateRegRegInstruction(cmp_op, node, reg->getHighOrder(), trgReg->getHighOrder(), cg);
-      generateLabelInstruction(isMin ? JG4 : JL4, node, doneLabel, false, cg);
-      generateLabelInstruction(JNE4, node, gtLabel, false, cg);
-      generateRegRegInstruction(cmp_op, node, reg->getLowOrder(), trgReg->getLowOrder(), cg);
-      generateLabelInstruction(isMin ? JA4 : JB4, node, doneLabel, false, cg);
-      generateLabelInstruction(LABEL, node, gtLabel, cg);
-      generateRegRegInstruction(move_op, node, trgReg->getHighOrder(), reg->getHighOrder(), cg);
-      generateRegRegInstruction(move_op, node, trgReg->getLowOrder(), reg->getLowOrder(), cg);
-
-      dep = new (cg->trHeapMemory()) TR::RegisterDependencyConditions(0, 4, cg->trMemory());
-      dep->addPostCondition(trgReg->getLowOrder(), TR::RealRegister::NoReg, cg);
-      dep->addPostCondition(trgReg->getHighOrder(), TR::RealRegister::NoReg, cg);
-      dep->addPostCondition(reg->getLowOrder(), TR::RealRegister::NoReg, cg);
-      dep->addPostCondition(reg->getHighOrder(), TR::RealRegister::NoReg, cg);
-      }
-   else
+static TR::Register *minmaxEvaluator64BitPrimitiveOnIA32(TR::Node *node, TR::CodeGenerator *cg)
+   {
+   TR_ASSERT(TR::Compiler->target.is32Bit(), "ONLY 32-BIT CODE SHOULD REACH HERE.");
+   TR_X86OpCodes SETcc = BADIA32Op;
+   switch (node->getOpCodeValue())
+      {
+      case TR::lmin:
+         SETcc = SETL1Reg;
+         break;
+      case TR::lmax:
+         SETcc = SETG1Reg;
+         break;
+      default:
+         TR_ASSERT(false, "INCORRECT IL OPCODE.");
+         break;
+      }
+   auto oprand0 = cg->evaluate(node->getChild(0));
```

Comment:
I'm not sure if its intentional to abbreviate `operand` by one letter or if its just a spelling mistake, but I think it reads better as `operand`.  Please change throughout this function.

Reply:
It was a typo. Now it has been fixed.